src/applications/piezoscan.py

```diff
@@ -339,8 +353,13 @@ def go_to_position(self, event = None):
             self.counter_scanner.stage_controller.go_to_position(x = self.view.sidepanel.go_to_x_position_text.get(), y = self.view.sidepanel.go_to_y_position_text.get())
         else:
             print(f'stage_controller would have moved to x,y = {self.view.sidepanel.go_to_x_position_text.get():.2f}, {self.view.sidepanel.go_to_y_position_text.get():.2f}')
-        self.optimized_position['x'] = self.view.sidepanel.go_to_x_position_text.get()
-        self.optimized_position['y'] = self.view.sidepanel.go_to_y_position_text.get()
+        x, y  = self.view.sidepanel.go_to_x_position_text.get(),  self.view.sidepanel.go_to_y_position_text.get()
+        self.optimized_position['x'] = x
+        self.optimized_position['y']  =y
+        self.view.scan_view.circle_pos = [x,y]
+        if len(self.counter_scanner.scanned_count_rate) > 0:
+            self.view.scan_view.update(self.counter_scanner)
+            self.view.canvas.draw()
 
```

Comment:
If the ScanImage object retains handles to the "pos" and "pointer" object, we might be able to update the plot by simply updating the data within those object. But I don't know... would have to test and see.

Reply:
done!